examples/flutter_bloc_with_stream/lib/ticker/ticker.dart

```diff
@@ -0,0 +1,6 @@
+import 'dart:async';
+
+class Ticker {
+  Stream<int> tick() =>
```

Comment:
nit but since this spans multiple lines can we rewrite like:

```dart
Stream<int> tick() {
return Stream.periodic(Duration(seconds: 1), (x) => x).take(10);
}
```